src/main/java/com/salesforce/dynamodbv2/mt/util/CachingAmazonDynamoDbStreams.java

```diff
@@ -81,13 +91,16 @@
         private static final int DEFAULT_MAX_GET_RECORDS_RETRIES = 10;
         private static final long DEFAULT_GET_RECORDS_LIMIT_EXCEEDED_BACKOFF_IN_MILLIS = 1000L;
         private static final int DEFAULT_MAX_ITERATOR_CACHE_SIZE = 100;
+        private static final long DEFAULT_DESCRIBE_STREAM_CACHE_TTL = 1;
```

Comment:
Should we default to a slightly higher value (like 5s)?

Reply:
updated in a032f5ee47ba57b46c150adadbf3552a1458a2af
Added a metric TODO for describeStream totalTime as this will help us adjust the default.